tests/baselines/reference/tsserver/fourslashServer/completionsImport_jsModuleExportsAssignment.js

```diff
@@ -1943,6 +1956,19 @@ Info seq  [hh:mm:ss:mss] response:
               "fileName": "/third_party/marked/src/defaults.js"
             }
           },
+          {
+            "name": "defaults",
+            "kind": "property",
+            "kindModifiers": "",
+            "sortText": "16",
+            "hasAction": true,
+            "source": "/third_party/marked/src/defaults",
+            "data": {
+              "exportName": "export=",
+              "exportMapKey": "8 * defaults ",
+              "fileName": "/third_party/marked/src/defaults.js"
+            }
```

Comment:
This was being filtered out by the `!(symbol.flags & SymbolFlags.Transient)` condition in utilities.ts that I removed here. That code path was no longer exercised by the name generation code in the ExportInfoMap construction, but I was curious why it was there, and traced it back to https://github.com/andrewbranch/TypeScript/commit/258be217a6244efcd8055d4096b50585427c5dce to see if it was going to be a problem. So much has changed around this in the last three years that it seems like the crash is no longer an issue, so I removed the transient symbol filtering from the other function where it was happening.